openshift/catalogd/kustomize/overlays/openshift/olmv1-ns/metrics/catalogd_metrics_monitor_role_binding.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: metrics-monitor-rolebinding
```

Comment:
Could we use a more intuitive name, like ocp-metrics-monitoring-rolebinding?

Reply:
already discussed here: https://github.com/openshift/operator-framework-operator-controller/pull/252#discussion_r1944285206